packages/stark-ui/src/modules/app-footer/components/app-footer.component.html

```diff
@@ -0,0 +1,27 @@
+<footer class="stark-app-footer-container">
+	<span translate>STARK.APP_FOOTER.COPYRIGHT</span>
+		&copy; 
+	<span>{{ copyrightPeriod }}</span>
+	<span class="bullet" *ngIf="legalInfoUrl">
+		&bull; 
+	</span>
+	<a *ngIf="legalInfoUrl"
+	href="{{ legalInfoUrl }}"
+	translate>
+		STARK.APP_FOOTER.LEGAL_INFO
+	</a>
+	<span class="bullet" *ngIf="helpPageUrl">
+		&bull;
+	</span>
+	<a *ngIf="helpPageUrl"
+	href="{{ helpPageUrl }}"
+	translate>
+		STARK.APP_FOOTER.HELP
+	</a>
+	<span class="bullet" *ngIf="footerContent.childNodes.length !== 0">
+		&bull;
+	</span>
+	<span #footerContent>
```

Comment:
@SuperITMan @christophercr Do we really need the footerContent? If yes, as we don't control what the end user place inside, I think the `.stark-app-footer-container` should be a flexbox where we center vertically all the children.

Reply:
Yes, we need the footerContent, and yes, I agree that it should be vertically aligned 😉 

@catlabs could you help @dede-pon to do this?